app/server/appsmith-interfaces/src/main/java/com/appsmith/external/helpers/restApiUtils/helpers/RestAPIActivateUtils.java

```diff
@@ -213,6 +225,10 @@ protected Mono<ClientResponse> httpCall(
                 });
     }
 
+    public URI createRedirectUrl(String redirectUrl, URI originalUrl) throws URISyntaxException {
```

Comment:
I don't think this exception is being thrown in here.

```suggestion
    public URI createRedirectUrl(String redirectUrl, URI originalUrl) {
```